modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/HorizontalCardTag.java

```diff
@@ -0,0 +1,383 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.taglib.clay.servlet.taglib;
+
+import com.liferay.frontend.taglib.clay.internal.servlet.taglib.BaseContainerTag;
+import com.liferay.frontend.taglib.clay.servlet.taglib.soy.HorizontalCard;
+import com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItem;
+import com.liferay.petra.string.StringPool;
+import com.liferay.portal.kernel.language.LanguageUtil;
+import com.liferay.portal.kernel.util.ListUtil;
+import com.liferay.taglib.util.TagResourceBundleUtil;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import javax.servlet.jsp.JspException;
+import javax.servlet.jsp.JspWriter;
+
+/**
+ * @author Marko Cikos
+ */
+public class HorizontalCardTag extends BaseContainerTag {
+
+	@Override
+	public int doStartTag() throws JspException {
+		setAttributeNamespace(_ATTRIBUTE_NAMESPACE);
+
+		Map<String, String> data = getData();
+
+		if (data != null) {
+			for (Map.Entry<String, String> entry : data.entrySet()) {
+				setDynamicAttribute(
+					StringPool.BLANK, "data-" + entry.getKey(),
+					entry.getValue());
+			}
+		}
+
+		return super.doStartTag();
+	}
+
+	public List<DropdownItem> getActionDropdownItems() {
+		if ((_actionDropdownItems != null) && (_horizontalCard != null)) {
```

Comment:
Shouldn't this be 

```java 
if ((_actionDropdownItems == null) && (_horizontalCard != null)) { }
```

As we did [here](https://github.com/liferay-frontend/liferay-portal/pull/452/files#diff-7df0e8ed3761415125433a5b6417c3f72c6b95449afa355c819a7cfccf796daeR44)

Reply:
Yes :+1: